defaults/main.yml

```diff
@@ -26,5 +39,7 @@ jenkins_plugins:
   - template-project
   - ec2
 
+jenkins_custom_plugins: []
```

Comment:
Can you update the README with details of how this and the other new variables work?